src/components/CreateAccount/CreateAccountContainer.jsx

```diff
@@ -1,15 +1,57 @@
 import React, { useState } from 'react';
+import RequestService from '../../util/RequestService'
 import './CreateAccountContainer.css';
 
 import ideaLABlogo from './../../ideaLABlogo.png';
 
 const LoginManager = () => {
```

Comment:
Let's change the name of the file, its references, and the const to `CreateAccountManager`. Login logic will be elsewhere.